crates/fluvio-dataplane-protocol/src/common.rs

```diff
@@ -21,6 +22,25 @@ impl Default for Isolation {
     }
 }
 
+impl FromStr for Isolation {
```

Comment:
Why put conversion in this crate?  Opinionated conversion (user interface) should be put in their respective area (CLI?)

Reply:
Due to Orphan rules, we can't impl `FromStr` anywhere else. We might put some custom conversion method in `fluvio-cli` but we may have needed it in other places in the future. Anyways, I don't have a strong opinion here, maybe just a method inside `fluvio-cli` crate is better for now. What would you recommend?

Reply:
For now, just a method in the `fluvio-cli` since that's only used there.  If need to make it more usable, you can use https://doc.rust-lang.org/rust-by-example/generics/new_types.html to get around orphan rules.

Reply:
Added parse function to `fluvio-cli` crate.